exercises/proverb/.meta/src/reference/java/Proverb.java

```diff
@@ -0,0 +1,27 @@
+final class Proverb {
+    private String[] words;
+    private String initWord, qualifier, result = "";
+
+    Proverb(String... words) {
```

Comment:
I would use `String[]`. Using `String...` adds some unnecessary complixity to the exercise and I don't think it's really needed :)